src/rgw/rgw_op.cc

```diff
@@ -715,6 +715,14 @@ int RGWGetObj::read_user_manifest_part(rgw_bucket& bucket, RGWObjEnt& ent, RGWAc
     return -EPERM;
   }
 
+  if (ent.size == 0) {
+    /* This is a 0 sized segment, which can be easily created in a DLO.
+     * Return now; we're done.
+     * (BZ #1380196)
```

Comment:
Can we please refer to a tracker.ceph.com ticket here, instead of Red Hat's Bugzilla?